README.md

```diff
@@ -49,363 +35,61 @@ Yomitan provides advanced features not available in other browser-based dictiona
 
-Online dictionaries will not be implemented because it is not possible to support them in a robust way. In order to
-perform Japanese deinflection, Yomitan must execute dozens of database queries for every single word. Factoring in
-network latency and the fragility of web scraping, it would not be possible to maintain a good and consistent user
-experience.
+    <img src="resources/images/settings-dictionaries-popup.png" width="300" alt="custom dictionaries list">
 
-**Is it possible to use Yomitan with files saved locally on my computer with Chrome?**
+3.  Webpage text can be scanned by moving the cursor while holding a modifier key, which is <kbd>Shift</kbd> by default. If definitions are found for the text at the cursor position, a popup window containing term definitions will open. This window can be dismissed by clicking anywhere outside of it.
 
-In order to use Yomitan with local files in Chrome, you must first tick the _Allow access to file URLs_ checkbox
-for Yomitan on the extensions page. Due to the restrictions placed on browser addons in the WebExtensions model, it
-will likely never be possible to use Yomitan with PDF files.
+    <img src="resources/images/search-popup-terms.png" width="300" alt="popup with search terms">
 
-**Is it possible to delete individual dictionaries without purging the database?**
+4.  Click on the <img src="ext/images/play-audio.svg" width="16" height="16" alt="loudspeaker icon"> _speaker_ button to hear the term pronounced by a native speaker. If an audio sample is not available, you will hear a short click instead. You can configure the sources used to retrieve audio samples in the options page.
 
-Yomitan is able to delete individual dictionaries, but keep in mind that this process can be _very_ slow and can
-cause the browser to become unresponsive. The time it takes to delete a single dictionary can sometimes be roughly
-the same as the time it originally took to import, which can be significant for certain large dictionaries.
+5.  Click on individual kanji in the term definition results to view additional information about those characters, including stroke order diagrams, readings, meanings, as well as other useful data.
 
-**Why aren't EPWING dictionaries bundled with Yomitan?**
+    <img src="resources/images/search-popup-kanji.png" width="300" alt="popup with kanji details">
 
-The vast majority of EPWING dictionaries are proprietary, so they are unfortunately not able to be included in
-this extension due to copyright reasons.
+To further enhance your Yomitan experience, it's worth [integrating with Anki](./docs/anki-integration.md), a spaced-repetition flashcard program to help solidify the words you encounter.
 
-**When are you going to add support for $MYLANGUAGE?**
+## Contributing
```

Comment:
I think I'd prefer this to be where it was originally, as this information is high priority (to me). We need to keep a very strong developer on-boarding pipeline for the project to stay alive.